tests/src/Functional/ContentProjectRenderTest.php

```diff
@@ -291,4 +320,102 @@ protected function getStorage(string $entity_type_id): EntityStorageInterface {
     return \Drupal::entityTypeManager()->getStorage($entity_type_id);
   }
 
+  /**
+   * Updates the project date, saves the node, and refreshes the page.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   Node to update.
+   * @param string $begin
+   *   Start date string.
+   * @param string $end
+   *   End date string.
+   */
+  protected function setProjectDateRange(NodeInterface $node, string $begin, string $end): void {
```

Comment:
```suggestion
  protected function setProjectDateRangeAndRefresh(NodeInterface $node, string $begin, string $end): void {
```

But honestly a function should do 1 thing and 1 thing only, so the AndSomething should already tell you its nok.
The test is already full of drupalGets, I dont see why it would be an issue to continue, its part of the tests stroy to see you revisit the page, right now its not clear that the page is revisited/refreshed